tests/testthat/test_environment.R

```diff
@@ -32,32 +32,24 @@ test_that("create dockerfile", {
   # cran packages
   dockerfile <- generate_docker_file(custom_docker_image = "ubuntu-18.04",
                                      cran_packages = c("ggplot2", "dplyr"))
-  expected_dockerfile <- paste0(
-    "FROM ubuntu-18.04\n",
-    "RUN R -e \"install.packages(\'ggplot2\', ",
-    "repos = \'http://cran.us.r-project.org\')\"\n",
-    "RUN R -e \"install.packages(\'dplyr\', ",
-    "repos = \'http://cran.us.r-project.org\')\"\n")
-  expect_equal(dockerfile, expected_dockerfile)
-  
+  expect_equal(dockerfile, "FROM ubuntu-18.04\n")
```

Comment:
We still can support incremental build with using a custom docker image as base image, right?

Reply:
yes right. made the changes